.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build & Test
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  build:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-2019, ubuntu-18.04 ]
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
+      - name: Setup .NET Core 3.1 runtime
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.x
+      - name: Restore
+        run: dotnet restore src/NServiceBus.sln --disable-parallel          
```

Comment:
Out of interest, why `--disable-parallel`?

Reply:
Another GitVersion Problem https://github.com/Particular/KeepTheLightsOn/blob/master/constraints/build-deployment-infrastructure/github-complex-actions.md#restore

Reply:
We have updated Particular.Packaging with the latest GitVersion. Fingers crossed